pkgs/by-name/do/dotbot/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+}:
+
+with python3Packages;
```

Comment:
Using `with` statements at the top level is not preferred.
This is because it makes it difficult to tell which variables come from python3Packages.
If necessary, it is better to restrict the scope as follows.

```nix
nativeBuildInputs = with python3Packages; [
  setuptools
];
```